extensions/Cake.DocCreator.yml

```diff
@@ -3,10 +3,11 @@ Name: Cake.DocCreator
 NuGet: Cake.DocCreator
 Assemblies:
 - "/**/Cake.DocCreator.dll"
-Repository: https://github.com/agc93/DocCreator/
+Repository: https://bitbucket.org/agc93/DocCreator/
```

Comment:
Why the change here?

Reply:
I haven't done the research but my guess is that the author moved the repo from bitbucket to github between June 2016 and April 2017. That would explain the difference between these two URLs.